client/settings/payment-settings/index.js

```diff
@@ -109,7 +113,11 @@ const AccountDetailsSection = () => {
 						{ accountStatus.email }
 					</h4>
 				) }
-				{ accountStatus.mode === 'test' && <Pill>Test Mode</Pill> }
```

Comment:
I noticed that some of these strings were not wrapped in `__(...)` - also modified to use the `useTestMode()` hook